modin/config/pubsub.py

```diff
@@ -74,6 +74,9 @@ class Parameter(object):
     type = str
     default = None
     is_abstract = True
+    # This flag is used to detect a way of setting value for an entity.
+    way_of_set = None
+    WAYS_OF_SET = {"default": 0, "from_config": 1, "from_user": 2}
```

Comment:
What is the difference between `from_user` and `from_config`? Can we document it?

Reply:
The difference is that `from_user` maps a value set by `put` and `from_config` maps a value set by env var. Probably, it makes sense to rename `from_user` to `put` and `from_config` to `envvar`. So, added comment on the dictionary and its content.